templates/modulesTakenPriorToInternship.html

```diff
@@ -0,0 +1,52 @@
+$def with (moduleListWithCount, list_of_ay_sems, ay_sem)
+
+$var title:View Modules Taken Prior to Internship
+
+<div class="container-fluid">
+	<h1 class="text-center"><b>Modules Taken Prior to Internship for $ay_sem</b></h1>
+	<p class="text-center">Shows all modules taken by students prior to taking internship for a particular semester, and how many students took them.</b> <span class="glyphicon glyphicon-info-sign" data-toggle="tooltip" data-placement="bottom" title="By default, this shows all data before students take internship in the current AY-Sem."></span></p>
+	<br>
+	<div class="row"><!--request form-->
+        <div class="col-md-12 pagination-centered text-center">
+            <form id="ay-form" class="form-inline" action="/moduleTakenPriorToInternship" method="post">
+                <div class="form-group">
+                    <label for="ay-sem">Select AY-Sem:</label>
+                    <select class="form-control" name="sem">
+                        <option value="" disabled selected hidden>Please choose a target AY-Semester</option>
+                        $for aysem in list_of_ay_sems:
+                        	<option value="$aysem">$aysem</option>
+                    </select>
+                </div>
+                <div class="form-group">
+                    <input type="submit"  value="Submit" class="btn btn-primary">
+                </div>
+            </form>
+        </div>
+    </div>
+	<br>
+
+	<table class="table display dataTable table-bordered table-hover text-center" id="modules-taken-prior-intern-table">
```

Comment:
![image](https://cloud.githubusercontent.com/assets/12386860/24032716/603f7d52-0b24-11e7-8e05-813286e23333.png)

Really? The sortings exist on my side.